src/main/java/com/github/johrstrom/listener/PrometheusListener.java

```diff
@@ -115,6 +116,11 @@ public void sampleOccurred(SampleEvent event) {
 			String[] samplerLabelValues = this.labelValues(event);
 			if (collectSamples)
```

Comment:
You'll need to extend the {} around this if block. It works without brackets only because there was 1 statement under it.